src/module.h

```diff
@@ -197,6 +198,9 @@ class Module {
     const char *filename;
     AST *ast;
 
+    std::string CPPBuffer;
+    llvm::raw_string_ostream CPPStream;
```

Comment:
> When we do regular compilation, the changes will make previously short-lived buffer (of preprocessed code), live till the Module is destroyed, i.e. during the whole compilation.
@dbabokin

Reply:
> I would probably make a llvm::raw_string_ostream not a member of Module, but a pointer to llvm::raw_string_ostream make a member of the Module and allocate it on the heap. This will require more sophisticated delete logic, though
@dbabokin